packages/nodes-base/nodes/Aws/SES/AwsSes.node.ts

```diff
@@ -846,7 +846,7 @@ export class AwsSes implements INodeType {
 						const params = [
 							'Action=CreateCustomVerificationEmailTemplate',
 							`FailureRedirectionURL=${failureRedirectionURL}`,
-							`FromEmailAddress=${email}`,
+							`FromEmailAddress=${encodeURIComponent(email)}`,
 							`SuccessRedirectionURL=${successRedirectionURL}`,
```

Comment:
do we not need to do this for `successRedirectionURL` as well?

Reply:
I guess it wouldn't hurt to add it to success and failure redirection urls, Thanks will do this now.